process/block/sovereign/incomingHeader/incomingEventsProc/eventProcDepositOperation_test.go

```diff
@@ -0,0 +1,29 @@
+package incomingEventsProc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewEventProcDepositOperation(t *testing.T) {
+	t.Parallel()
+
+	t.Run("nil deposit event proc, should return error", func(t *testing.T) {
+		handler, err := NewEventProcDepositOperation(nil, &eventProcScCall{})
+		require.Equal(t, errNilEventProcDepositTokens, err)
+		require.Nil(t, handler)
+	})
+
+	t.Run("nil sc call event proc, should return error", func(t *testing.T) {
+		handler, err := NewEventProcDepositOperation(&eventProcDepositTokens{}, nil)
+		require.Equal(t, errNilEventProcScCall, err)
+		require.Nil(t, handler)
+	})
+
+	t.Run("should work", func(t *testing.T) {
+		handler, err := NewEventProcDepositOperation(&eventProcDepositTokens{}, &eventProcScCall{&eventProcDepositTokens{}})
+		require.NotNil(t, handler)
+		require.Nil(t, err)
```

Comment:
Just for code coverage, can you also add a check false for `IsInterfaceNil` here?